integration-tests/vertx/pom.xml

```diff
@@ -27,6 +27,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-undertow</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Undertow comes with the resteasy extension (and it shouldn't be provided anyway).

Reply:
Not sure what you mean. Without it the tests did not run.

Reply:
Just to be clear: this dependency should be dropped.

Reply:
Let me take a look at it.

Reply:
I confirm it works without it with your PR on top of master and a clean build.

It should definitely be removed.

Reply:
Just rebased and updated.